feature-libs/storefinder/styles/components/_store-finder-map.scss

```diff
@@ -3,9 +3,19 @@
     width: 100%;
     height: 100%;
 
-    @include media-breakpoint-down(sm) {
-      height: 30rem;
-      width: 100%;
+    // TODO:  Remove feature flag next major
```

Comment:
Really just a nitpick to add the cleanup ticket number. It probably won't be you doing this cleanup in the future so it really helps ;)

Reply:
Done :)